src/components/BrewerySearch.js

```diff
@@ -1,21 +1,30 @@
 import React, { Component } from 'react'
 import Typed from 'react-typed'
 import RadioButtonContainer from './RadioButtonContainer'
+import Modal from 'react-bootstrap/Modal'
 
 export default class BrewerySearch extends Component {
   state = {
     search: '',
-    searchBy: 'city'
-  }
+    searchBy: 'city',
+    show: false
+  };
 
   updateSearch = event => {
     this.setState({ search: event.target.value })
   }
 
   onSearchClick = e => {
     const { search } = this.state
-    if (search) {
-      this.props.handleSearch(search)
+
+    if(search.length === 0){
```

Comment:
Hmmm... the modal isn't a bad idea, but it is kinda obtrusive as you can't just click or tab out of it to close it. I'm sure react-bootstrap Modals already have this accessibility feature? It also kinds of blends into the dark background of the cover image. 

A suggestion I want to make to replace the Modal is with an `Alert` component. Alert already has warning, danger, etc color using its variant prop. Additionally we can have it fade out on its own after a few second. 

Here's a [sandbox](https://codesandbox.io/s/n5r2qw0q3m?fontsize=14) of what I mean. I think this route would be easier to interact with and visually appear better. Also now we can start using React's hooks. 

What do you guys think?
// @JasonFritsche  @PabloRosas17

Reply:
@cdrani I agree, this implementation could be made better by using an `Alert`. Also, how do you feel about a message that is displayed for a set amount of seconds only? 
Thanks @cdrani && @PabloRosas17

Reply:
Check the sandbox. Press enter with empty search and the alert appears and then disappears after two seconds.

Reply:
i've ran the sandbox, i think it's a fantastic idea. although i did find the pop up modal a bit attractive and can see some room for growth on it. i was thinking, do the sandbox first as it is the cleanest and instant solution. however, for the future i'd like to reference this issue by going the modal route. i'm thinking if we use a popup modal due to an empty search, perhaps we can create a custom suggestions search.

something like this:
Oops! No empty searches allowed, but check out some of our recommended (pubs/bars/wineries) in your area.

Reply:
for now i will rework this issue using an alert, does that work with the team? 
please also let's create an issue for my suggestion above if you like what i'm saying,
should i include a diagram to illustrate my suggestion in an abstract form? will it help?

Reply:
1) Yes, full agreement on Alert all around. Please proceed.
2) I'm not sold on the Modal. I still think it's unneeded - especially anywhere near a search form.
   * What if someone just forgets to enter input? Is the modal supposed to show up every time? That would be annoying and distracting having to interact with it each time. Recommendations should be passive not aggressive.
   * We don't need a modal to showcase recommended places. This can just be added anywhere in below the search form.
   * The api is America centralized, therefore the location based recommendations would only benefit people already in the area.
   * However, I can see adding recommendations in a #9 (details page) that would be based on what the user has just searched. 

3) I'm not the one to be convinced anyways, these are just my opinions. I would still love to see the diagram.

Reply:
@cdrani I agree with your arguments here. @PabloRosas17 Let's go forward without the modal. Thanks for your efforts. I look forward to seeing your work.

Reply:
your explanation is clarifying @cdrani , let's toss the modal aside.
sounds good @JasonFritsche , i'll spectate for now.

Reply:
@PabloRosas17: Are you working on this currently? We could free this up to someone else or I could complete it if you don't have time. Just let us know. Thanks.